app/helpers/application_helper.rb

```diff
@@ -126,7 +126,7 @@ def non_anonymous_byline(creation)
   end
 
   def pseud_link(pseud)
```

Comment:
This is used in `byline`, which is used in a lot of places, some of which need the full URL -- have you checked RSS feeds, downloads, emails, and share code to make sure that they still have the full URL? I just did a spot check and while the share code looks okay, an HTML download of a work has a broken link for the byline, giving me only `<a href="/users/testy2/pseuds/testy2" rel="author">testy2</a>` when it needs to be the full URL.